docker/otel-collector/log-rotator.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
```

Comment:
running the log rotator script along with the main otelcol process to rotate `agent.log` file since its size grows rapidly